### ISSUE TITLE
Plots optimization

### ISSUE THREAD

**Issue by IngvarJackal** [Fenrikur/Nosecounter#1]
It might be better, if
 * log-shaped plots were actually be log-transformed by Y axis (Registrations per 1 Minute, 
Attendance by Country)
 * Attendance by Status was a piechart instead of a bar plot
 * Comparison plots were grouped stacked normalized column plots like Attendance by Gender (Comparison) -- Attendance by Country (Comparison) could be changed, for example.

**Comment by Banane9** [comment]
It'd also be nice if all the countries on the "Attendance by Country" graph would be labeled

![Attendance by Country](http://puu.sh/x7sK4/aa1a28f622.png)

Because right now, all the bars except for the labeled ones are... well, useless.

Also, it might be good to make this a logarithmic plot as well, to give more detail where it matters.

**Comment by Fenrikur** [comment]
> - log-shaped plots were actually be log-transformed by Y axis (Registrations per 1 Minute,
Attendance by Country)
> - Attendance by Status was a piechart instead of a bar plot

Those are definitely good points I will try to look into before launching this year's edition of the Nosecounter!

> - Comparison plots were grouped stacked normalized column plots like Attendance by Gender (Comparison) -- Attendance by Country (Comparison) could be changed, for example.

I tried to use normalised stacked column plots wherever I deemed them to be useful and not lead to false assumptions. I will evaluate how it works out for the Attendance by Country (Comparison) plot, also compared to using a log scale for that one.
For some other charts, like the demographics comparison, using normalised stacks will not work, as every attendee may be part of any, all or none of the demographic groups listed there, making them non-disjoint sets. Stacking them would therefore be wrong, as their sum would not hold any useful information, with its maximum value being the number of attendees multiplied by the number of available demographic groups.
I am still looking for a nicer visualisation of the Age Distribution (Comparison) chart, for which I formerly used stacks, but found the current plot to be more useful. Applying a log scale to the y-axis of that one as well should at least be of some help though.

> It'd also be nice if all the countries on the "Attendance by Country" graph would be labeled

This is an actual bug, that I will work on fixing before this year's reg opens.

Thanks again for your feedback, it's always welcome!
